sql/moz-fx-data-shared-prod/telemetry/applications/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.applications`
```

Comment:
Nit/observation: `telemetry` dataset becomes a bit overloaded, historically it was for legacy telemetry.

Reply:
Yeah, I noticed that too... It's basically become the default dataset to use when there is no better dataset. There might be some opportunity to break things up into smaller datasets